packages/gestalt/src/InternalLink.js

```diff
@@ -224,10 +226,10 @@ const InternalLinkWithForwardRef: AbstractComponent<
         ...(target === 'blank' ? ['noopener', 'noreferrer'] : []),
         ...(rel === 'nofollow' ? ['nofollow'] : []),
       ].join(' ')}
+      tabIndex={removeFromSequentialKeyboardNavigation ? '-1' : '0'}
       {...(tapStyle === 'compress' && compressStyle && !disabled
         ? { style: compressStyle }
         : {})}
-      tabIndex={isTapArea && !disabled ? '0' : null}
```

Comment:
the default is always 0.

Reply:
@AlbertCarreras I think this change is wrong, when a TapArea is disabled, the `tabIndex` should be `null` or `-1`

```
<TapArea
        role="link"
        target="blank"
        href="https://www.pinterest.com"
        disabled
      >
```

![image](https://user-images.githubusercontent.com/127199/95227677-3e423980-07b3-11eb-95e6-44fe94e604b1.png)

Reply:
MMM I read the in absence of a tabIndex, the default is 0. I'll double check 

[If not specified, it takes the default value 0.](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

Reply:
@AlbertCarreras that is true for elements which are already focussable (e.g. `<a />`, `<button />`) but not for others (e.g. `<div />`)